facter reports wrong swap properties. Solaris specific patch.
No upstream. facter 2.x is no longer supported and new bugs are not being accepted.
-----------------------------------------------------------------------------------
--- facter-2.5.1/lib/facter/util/memory.rb.orig	2020-10-13 23:25:18.002159879 +0000
+++ facter-2.5.1/lib/facter/util/memory.rb	2020-10-13 23:27:05.962978626 +0000
@@ -205,7 +205,7 @@
         0
       end
     when /SunOS/i
-      if line =~ /^\S+\s.*\s+(\d+)\s+(\d+)$/
+      if line =~ /^\S+\s.*\s+(\d+)\s+(\d+)\s+\S+$/
         (is_size) ? $1.to_i : $2.to_i
       else
         0
